api/swagger/server/restapi/resource/swagger.yml

```diff
@@ -618,6 +618,70 @@ paths:
           description: Internal Server Error - The server has encountered a situation it does not know how to handle.
           schema:
             $ref: "#/definitions/Error"
+  /cmd/deploySC:
+    put:
```

Comment:
I think it should be a post. Why ? Because imho Post is to be used for none idempotent action. here if you call it multiple times you will deploy the SC multiple times.